org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -207,6 +207,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
 
         dim.id = 'dim';
         dim.dimmingSetting = providerSchema['attributes'].level;
+        if (dim.dimmingSetting > 100 ) dim.dimmingSetting = 100;
```

Comment:
shouldn't we do a conversion here?  meaning what if its 125, wouldn't we want to set it to 50 in that case?  That requires us to guess, do we have data to back up that guess?